### ISSUE TITLE
language fr + show-inheritance = Constructor parameters bug

### ISSUE THREAD

**Issue by luc2** [sphinx-doc/sphinx#2968]
I get this:

![bug](https://cloud.githubusercontent.com/assets/2410567/18619548/b45e9f7e-7dff-11e6-90b9-96cda5db73e4.png)

Instead of that:

![expected](https://cloud.githubusercontent.com/assets/2410567/18619550/bdff74fe-7dff-11e6-900c-d348522d4d30.png)

I tried on Windows and CentOS, same bug.

**How to reproduce it on CentOS:**

``` bash
$ cat /etc/system-release
CentOS Linux release 7.1.1503 (Core)
$ sudo yum install https://centos7.iuscommunity.org/ius-release.rpm
$ sudo yum install python35u-pip
$ sudo pip3.5 install sphinx
$ mkdir myproject
$ cd myproject
$ vim mymodule.py
```

``` python
class MyClass(object):
    """My docstring:

    :param a: first parameter
    :param b: second
    :param c: third
    """
    def __init__(self, a, b, c):
        pass
```

``` bash
$ sphinx-quickstart
...
> Project language [en]: fr
...
> autodoc: automatically insert docstrings from modules (y/n) [n]: y
...
$ vim conf.py
```

``` python
...
import os
import sys
sys.path.insert(0, os.path.abspath('.'))
...
```

``` bash
$ vim index.rst
```

``` rst
...
.. automodule:: mymodule
   :members:
   :undoc-members:
   :show-inheritance:
...
```

``` bash
$ make html
$ echo Open with a web browser: _build/html/index.html
```

**Comment by samuel-phan** [comment]
Which version of Sphinx do you use?

**Comment by luc2** [comment]
``` bash
$ python3.5
>>> import sphinx
>>> sphinx.__version__
'1.4.6'
```

**Comment by tk0miya** [comment]
Thank you for reporting.
Fixed at    83d6ec1.
